tests/e2e/specs/modules/adsense/setup-new-user.test.js

```diff
@@ -0,0 +1,283 @@
+						message: "Your ad units are not displaying ads because you haven't provided your account payments information yet.",
+						severity: 'SEVERE',
+						type: 'BILLINGLESS_ACCOUNT',
+					},
+					{
+						id: 'GRAYLISTED_PUBLISHER',
+						isDismissible: false,
+						kind: 'adsense#alert',
+						message: 'Your AdSense application is still under review. You will only see blank ads until your account has been fully approved or disapproved.',
+						severity: 'SEVERE',
+						type: 'GRAYLISTED_PUBLISHER',
+					},
+					{
+						id: 'ALERT_TYPE_GLOBAL_BETTER_ADS_STANDARD',
+						isDismissible: false,
+						kind: 'adsense#alert',
+						message: "Global Better Ads Standards. Google Chrome will support the Better Ads Standards globally from July 9th. Ads may be filtered on Chrome browsers if you don't comply with the standard.",
+						severity: 'INFO',
+						type: 'ALERT_TYPE_GLOBAL_BETTER_ADS_STANDARD',
+					},
+				] ),
+			} );
+		};
+
+		await expect( '/' ).not.toHaveAdsenseTag();
+
+		await proceedToAdsenseSetup();
+
+		await expect( page ).toMatchElement( '.googlesitekit-setup-module__title', { text: /We’re getting your site ready for ads/i } );
+		await expect( page ).toMatchElement( '.googlesitekit-cta-link', { text: /Go to your AdSense account to check on your site’s status/i } );
+
+		await expect( '/' ).not.toHaveAdsenseTag();
+	} );
+
+	it( 'displays “We’re getting your site ready for ads” when the Adsense account is missing the address or phone not verified', async () => {
+		datapointHandlers.accounts = ( request ) => {
+			request.respond( {
+				status: 200,
+				body: JSON.stringify( [
+					ADSENSE_ACCOUNT,
+				] ),
+			} );
+		};
+		datapointHandlers.alerts = ( request ) => {
+			request.respond( {
+				status: 500,
+				body: JSON.stringify( {
+					code: 403,
+					message: {
+						error: {
+							errors: [
+								{
+									domain: 'global',
+									reason: 'accountPendingReview',
+									message: 'Users account is pending review.',
+								},
+							],
+							code: 403,
+							message: 'Users account is pending review.',
+						},
+					},
+					data: {
+						status: 500,
+					},
+				} ),
+			} );
+		};
+		datapointHandlers.clients = ( request ) => {
+			request.respond( {
+				status: 200,
+				body: JSON.stringify( [
+					{
+						arcOptIn: false,
+						id: `ca-${ ADSENSE_ACCOUNT.id }`,
+						kind: 'adsense#adClient',
+						productCode: 'AFC',
+						supportsReporting: true,
+					},
+				] ),
+			} );
+		};
+
+		await expect( '/' ).not.toHaveAdsenseTag();
+
+		await proceedToAdsenseSetup();
+
+		await expect( page ).toMatchElement( '.googlesitekit-setup-module__title', { text: /We’re getting your site ready for ads/i } );
+		await expect( page ).toMatchElement( '.googlesitekit-cta-link', { text: /Go to your AdSense account to check on your site’s status/i } );
+
+		// Note this _should_ output a tag but currently does not as `useSnippet` is not defaulting to true.
```

Comment:
This is a bit unexpected. I assumed an AdSense snippet would be displayed after setup is complete. But well 🤷‍♂

Reply:
Same. See #556 😄